Pod/Classes/KolodaView/KolodaView.swift

```diff
@@ -276,50 +223,18 @@ public class KolodaView: UIView, DraggableCardDelegate {
     }
     
     //MARK: Animations
-    public func applyAppearAnimation() {
-        userInteractionEnabled = false
-        animating = true
-        
-        let kolodaAppearScaleAnimation = POPBasicAnimation(propertyNamed: kPOPLayerScaleXY)
-        
-        kolodaAppearScaleAnimation.beginTime = CACurrentMediaTime() + cardSwipeActionAnimationDuration
-        kolodaAppearScaleAnimation.duration = kolodaAppearScaleAnimationDuration
-        kolodaAppearScaleAnimation.fromValue = NSValue(CGPoint: kolodaAppearScaleAnimationFromValue)
-        kolodaAppearScaleAnimation.toValue = NSValue(CGPoint: kolodaAppearScaleAnimationToValue)
-        kolodaAppearScaleAnimation.completionBlock = {
-            (_, _) in
-            
-            self.userInteractionEnabled = true
-            self.animating = false
-        }
-        
-        let kolodaAppearAlphaAnimation = POPBasicAnimation(propertyNamed: kPOPViewAlpha)
-        
-        kolodaAppearAlphaAnimation.beginTime = CACurrentMediaTime() + cardSwipeActionAnimationDuration
-        kolodaAppearAlphaAnimation.fromValue = NSNumber(float: Float(kolodaAppearAlphaAnimationFromValue))
-        kolodaAppearAlphaAnimation.toValue = NSNumber(float: Float(kolodaAppearAlphaAnimationToValue))
-        kolodaAppearAlphaAnimation.duration = kolodaAppearAlphaAnimationDuration
-        
-        pop_addAnimation(kolodaAppearAlphaAnimation, forKey: kolodaAppearAlphaAnimationName)
-        layer.pop_addAnimation(kolodaAppearScaleAnimation, forKey: kolodaAppearScaleAnimationName)
-    }
     
-    func applyRevertAnimation(card: DraggableCardView, complete: (() -> Void)? = nil) {
-        animating = true
-        
-        let firstCardAppearAnimation = POPBasicAnimation(propertyNamed: kPOPViewAlpha)
-        
-        firstCardAppearAnimation.toValue = NSNumber(float: Float(revertCardAnimationToValue))
-        firstCardAppearAnimation.fromValue =  NSNumber(float: Float(revertCardAnimationFromValue))
-        firstCardAppearAnimation.duration = revertCardAnimationDuration
-        firstCardAppearAnimation.completionBlock = {
-            (_, _) in
+    public func applyAppearAnimationIfNeeded() {
+        if let shouldApply = delegate?.koloda(kolodaShouldApplyAppearAnimation: self) where shouldApply == true {
```

Comment:
func applyAppearAnimationIfNeeded() {
   if ... { applyAppearAnimation }
}

func applyAppearAnimation() {
}

similarly to UIView design